pylint/testutils/_primer/primer_command.py

```diff
@@ -9,7 +9,7 @@
 from pathlib import Path
 from typing import Dict, TypedDict
 
-from pylint.reporters.json_reporter import OldJsonExport
+from pylint.reporters._json_reporter import OldJsonExport
```

Comment:
Good, I'm glad we're not immediately updating the primer to use json2. I have a toy branch that has a ShortJSONReporter that only captures the absolute minimum for the compare command to reduce file size.

Reply:
👍 

Shall we open an issue to do so? It should be trivial to change your `ShortJSONReporter` to use `JSONReporterTwo` 😄

Reply:
Oh, but I would want to keep the separate/private tiny one. (This one's too bulky).

Reply:
But wouldn't you just need to use `"messages"` and then be done?

Reply:
I was slicing away parts of the message, just keeping the necessary bits. Sometimes the message text is quite bulky.